### ISSUE TITLE
Low mAP on trainset

### ISSUE THREAD

**Issue by Axxeption** [fizyr/keras-retinanet#770]
Hi!

I made my own dataset to classify images using my own Pascal VOC-files. This dataset consists of 1200 pictures (600 of class 1 and 600 of class 2).

I just want to classify between 2 things, but my mAP doesn't get better then 0.61, even when I train 100 epochs. I would suppose that my network starts overfitting if I give a lot of epochs the same data, but it isn't. In addition my performance on my test set is very bad: the model almost always predicts the same one. 

I do not get this: is this maybe because there is some preprocessing so it't not possible to achieve better results (even on the train set)?

**Comment by hgaiser** [comment]
You can check debug.py to inspect your dataset to see if the data appears correct. Also I assume you're using the evaluation during training? It could also simply be a too complex dataset.

Regardless, I'm closing this as it is not an issue with the code. Please use Slack for usage questions.
